int32 map_id
string map_name
int32 task_id
int32 rate

---
float64[] data
string errormsg
bool  successed